site/docs/reference/Connectors/materialization-connectors/Rockset.md

```diff
@@ -12,9 +13,11 @@ To use this connector, you'll need :
     * Optional; if none exist, one will be created by the connector.
 
 ## Configuration
+
 To use this connector, begin with a Flow catalog that has at least one **collection**. You'll add a Rockset materialization, which will direct one or more of your Flow collections to your desired Rockset collections. Follow the basic [materialization setup](../../../concepts/materialization.md) and add the required Rockset configuration values per the table below.
```

Comment:
uber nit, but I'll ask just to get your thoughts on it: should we have the `**collection**` here be a link to our docs on Flow collections or how to create them?

Reply:
That's a good question. Gave it some thought, and I think it is safe to assume that by the time the reader gets to this point, they'll know what a collection is. Just by nature of the pathway, they'll have already created a capture and be familiar from that perspective. Plus it's such a high-level concept that if we linked it in every doc it's mentioned that could get link-heavy